shared/src/main/scala/ru/org/codingteam/euclid/IView.scala

```diff
@@ -0,0 +1,10 @@
+package ru.org.codingteam.euclid
+
+import ru.org.codingteam.euclid.io.{Display, KeyboardEvent}
+
+trait IView[TKeyCode] {
```

Comment:
> `IView`

> Scala

C# poisoned you buddy.

Reply:
I know that, and have already changed some of the original interface names to not contain that `I`.